dev.Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Use Nvidia's latest Ubuntu 22.04 image as the base image
+FROM nvidia/cuda:12.2.0-devel-ubuntu22.04
```

Comment:
```suggestion
# Use the official Python image as the base image
FROM python:3.11-slim-buster
```

I think we should use the python image for local development! Thanks for testing & flagging this 👍🏾 .